cookbooks/add_memory_using_qdrant_cloud.py

```diff
@@ -0,0 +1,41 @@
+# This example shows how to use vector config to use QDRANT CLOUD
+import os
+from dotenv import load_dotenv
+from mem0 import Memory
+
+# Loading my API_KEY for OPENAI
+load_dotenv()
+OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
+USER_ID = "test"
+os.environ['OPENAI_API_KEY'] = OPENAI_API_KEY
+quadrant_host="xx.gcp.cloud.qdrant.io"
+
+# creating the config attributes
+collection_name="memory" # this is the collection I created in QDRABT cloud
+api_key=os.environ.get("QDRANT_API_KEY") # Getting the QDRANT api KEY
+host=quadrant_host
+port=6333 #Default port for QDRANT cloud
+
+# Creating the config dict
+config = {
+    "vector_store": {
+        "provider": "qdrant",
+        "config": {
+            "collection_name": "memory",
```

Comment:
Also can you please use the variable that you have defined on line 14

Reply:
done